src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.SymbolToProjectId.cs

```diff
@@ -10,15 +10,15 @@
 
 namespace Microsoft.CodeAnalysis
 {
-    internal partial class SolutionState
+    internal partial class SolutionCompilationState
     {
         /// <inheritdoc cref="Solution.GetOriginatingProjectId"/>
-        public ProjectId? GetOriginatingProjectId(ISymbol? symbol)
+        public ProjectId? GetOriginatingProjectId(Solution solution, ISymbol? symbol)
```

Comment:
Why did this need a full Solution now versus the SolutionState that's already held? It looks like it's only grabbing DocumentStates in the first place with this, looking at where the this -> solution edits happened in this file?

Reply:
i'll double check.  likely something wonky as this refactored around.

Reply:
Yup.  Cleaned this all up.

Reply:
Yup.  Cleaned this up.  No need to pass solution anymore.